packages/server/graphql/mutations/deleteUser.ts

```diff
@@ -57,7 +59,9 @@ export default {
     } else if (!user) {
       return {error: {message: 'User not found'}}
     }
-    const {id: userIdToDelete} = user
+    const {id: userIdToDelete, tms} = user
+    const teamMemberIds = tms.map((teamId) => toTeamMemberId(teamId, userIdToDelete))
+    removeSlackAuths(teamMemberIds, true)
```

Comment:
Shouldn't need to `await` as the Slack logic isn't reliant on the subsequent logic